README.md

```diff
@@ -1 +1,13 @@
-# .github
\ No newline at end of file
+<p align="center" >
+<img src="https://docs.dash.org/en/stable/_images/dash_logo.png" alt="Dash Wallet logo" title="Dash Wallet" width="300">
+</p>
+
+IMPORTANT NOTE!!
+
+The Dashevo GitHub repo has been archived. The repos have all been relocated to https://github.com/dashpay
+
+Previously, all development related to Dash Platform happened under the dashevo Github org. These repositories have since moved to the DashPay repo. This was done to consolidate all the Platform-related activity under one platform.
```

Comment:
```suggestion
Previously, all development related to Dash Platform happened under the dashevo GitHub organization. These repositories have since moved to the DashPay organization. This was done to consolidate all Dash-related activity under one organization.
```